CMakeLists.txt

```diff
@@ -61,7 +62,7 @@ set(savitar_HDRS
     src/MeshData.h
     src/Vertex.h
     src/Face.h
-    src/SavitarExport.h
+    ${CMAKE_CURRENT_BINARY_DIR}/src/ArcusExport.h
```

Comment:
Oh, this should be `SavitarExport.h` I suppose. It's failing to install for me. I'll change it.

Reply:
Copy-pasta error - sorry!